internal/command/redis/create.go

```diff
@@ -30,6 +30,27 @@ func newCreate() (cmd *cobra.Command) {
 	flag.Add(cmd,
 		flag.Org(),
 		flag.Region(),
+		flag.String{
+			Name:        "name",
+			Shorthand:   "n",
+			Description: "The name of your Redis cluster",
+		},
+		flag.Bool{
+			Name:        "no-replicas",
+			Description: "No replica regions",
+		},
+		flag.Bool{
+			Name:        "enable-eviction",
+			Description: "Evict objects when memory is full",
+		},
+		flag.Bool{
```

Comment:
What's the purpose of this flag if 'disabled' is the default?

Reply:
Default actually is prompt.  If you specify either disabled or enabled you won't get prompted.